homeassistant/components/verisure/config_flow.py

```diff
@@ -0,0 +1,173 @@
+                    for inst in verisure.installations
+                }
+
+                return await self.async_step_installation()
+
+        return self.async_show_form(
+            step_id="user",
+            data_schema=vol.Schema(
+                {
+                    vol.Required(CONF_EMAIL): str,
+                    vol.Required(CONF_PASSWORD): str,
+                }
+            ),
+            errors=errors,
+        )
+
+    async def async_step_installation(
+        self, user_input: dict[str, Any] | None = None
+    ) -> dict[str, Any]:
+        """Select Verisure installation to add."""
+        if len(self.installations) == 1:
+            user_input = {CONF_GIID: list(self.installations)[0]}
+        elif self.giid and self.giid in self.installations:
+            user_input = {CONF_GIID: self.giid}
+
+        if user_input is None:
+            return self.async_show_form(
+                step_id="installation",
+                data_schema=vol.Schema(
+                    {vol.Required(CONF_GIID): vol.In(self.installations)}
+                ),
+            )
+
+        await self.async_set_unique_id(user_input[CONF_GIID])
+        self._abort_if_unique_id_configured()
+
+        return self.async_create_entry(
+            title=self.installations[user_input[CONF_GIID]],
+            data={
+                CONF_EMAIL: self.email,
+                CONF_PASSWORD: self.password,
+                CONF_GIID: user_input[CONF_GIID],
+                **self.settings,
+            },
+        )
+
+    async def async_step_import(self, user_input: dict[str, Any]) -> dict[str, Any]:
+        """Import Verisure YAML configuration."""
+        if user_input[CONF_GIID]:
+            self.giid = user_input[CONF_GIID]
+            await self.async_set_unique_id(self.giid)
+            self._abort_if_unique_id_configured()
+        else:
+            # The old YAML configuration could handle 1 single Verisure instance.
+            # Therefore, if we don't know the GIID, we can use the discovery
+            # without a unique ID logic, to prevent re-import/discovery.
+            await self._async_handle_discovery_without_unique_id()
+
+        # Settings, later to be converted to config entry options
+        if user_input[CONF_LOCK_CODE_DIGITS]:
+            self.settings[CONF_LOCK_CODE_DIGITS] = user_input[CONF_LOCK_CODE_DIGITS]
+        if user_input[CONF_LOCK_DEFAULT_CODE]:
+            self.settings[CONF_LOCK_DEFAULT_CODE] = user_input[CONF_LOCK_DEFAULT_CODE]
+
+        return await self.async_step_user(user_input)
+
+
+class VerisureOptionsFlowHandler(OptionsFlow):
+    """Handle Verisure options."""
+
+    def __init__(self, entry: ConfigEntry) -> None:
+        """Initialize Verisure options flow."""
+        self.entry = entry
+
+    async def async_step_init(
+        self, user_input: dict[str, Any] | None = None
+    ) -> dict[str, Any]:
+        """Manage Hue options."""
```

Comment:
Probably not Hue :)

Reply:
Hue was the last option flow I created, so bad copy & paste :D